packages/fe-fpm-writer/test/unit/__snapshots__/header-section.test.ts.snap

```diff
@@ -0,0 +1,403 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CustomHeaderSection generateCustomHeaderSection Positions Test 'position' property. Create with anchor - 1.85 ui5 1`] = `
+Object {
+  "flexSettings": Object {
+    "designtime": "Default",
+  },
+  "name": "sapux.fe.fpm.writer.test.extensions.custom.NewCustomHeaderSection",
+  "position": Object {
+    "anchor": "Dummy",
+    "placement": "Before",
+  },
+  "requestGroupId": "Decoration",
+  "subTitle": "Custom Header section subtitle",
+  "template": "sapux.fe.fpm.writer.test.extensions.custom.NewCustomHeaderSection",
+  "templateEdit": ".NewCustomHeaderSectionEdit",
```

Comment:
same issue like mentioned above in intergration test snapshot. Looks like wrongly generated entry for `templateEdit` - it does not have namespace before fragment name

Reply:
fix - 4c6fcdda05fc483345b76f297408a89e82084633